pkgs/servers/nosql/janusgraph/unwrapped.nix

```
@@ -0,0 +1,93 @@
+{ stdenv
+, lib
+, maven
+, jdk
+, protobuf
+, protoc-gen-grpc-java
+, unzip
+, version
+, src
+, srcIsBinary ? false
+, outputHash ? null
```

```suggestion
, outputHash ? ""
```

---

```suggestion
, outputHash
```

if it must be set then we should not set a default value